common.sh

```diff
@@ -1,20 +1,27 @@
 #!/bin/bash
 
-export PATH="/usr/local/go/bin:$HOME/.local/bin:$PATH"
+if ! [[ "PATH" =~ "$(systemd-path user-binaries)" ]]; then
+    PATH="$(systemd-path user-binaries):$PATH"
+fi
+if ! [[ "PATH" =~ "/usr/local/go/bin" ]]; then
+    PATH="/usr/local/go/bin:$PATH"
+fi
+export PATH
 
 # Set a PS4 value which logs the script name and line #.
 export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
-eval "$(go env)"
-
-export PATH="${GOPATH}/bin:$PATH"
-
-# Ensure if a go program crashes we get a coredump
-#
-# To get the dump, use coredumpctl:
-#   coredumpctl -o oc.coredump dump /usr/local/bin/oc
-#
-export GOTRACEBACK=crash
+if command -v go; then
+    eval "$(go env)"
+    export PATH="${GOPATH}/bin:$PATH"
+    # Ensure if a go program crashes we get a coredump
+    #
+    # To get the dump, use coredumpctl:
+    #   coredumpctl -o oc.coredump dump /usr/local/bin/oc
+    #
+    export GOTRACEBACK=crash
+    export OPENSHIFT_INSTALL_PATH="$GOPATH/src/github.com/openshift/installer"
```

Comment:
Isn't already set in [common.sh](https://github.com/openshift-metal3/dev-scripts/blob/081b22db68430997dd7a416114213d87c1c751bb/common.sh#L140)?

Reply:
mmm. 22-23 should not be in the PR. I don't know how they sneaked in there